src/measure_theory/measure/measure_space.lean

```diff
@@ -1624,18 +1624,22 @@ begin
   apply measure_mono_null (inter_subset_left t s) ht2
 end
 
-lemma _root_.set.countable.measure_zero (h : countable s) : μ s = 0 :=
+lemma _root_.set.countable.measure_zero {α : Type*} [measurable_space α] {s : set α}
+  (h : countable s) (μ : measure α) [has_no_atoms μ] :
```

Comment:
```suggestion
lemma _root_.set.countable.measure_zero {α : Type*} {m : measurable_space α} {s : set α}
  (h : countable s) (μ : measure α) [has_no_atoms μ] :
```
could you replace `[measurable_space α] (μ : measure α)` by `{m : measurable_space α} (μ : measure α)` whenever the measure is explicit? It makes sure that when you use the lemma, the measurable space structure inferred is the same as the one used to define the measure. See #8571 for details.